nidirect_breadcrumbs/src/HealthConditionBreadcrumb.php

```diff
@@ -76,15 +88,18 @@ public function build(RouteMatchInterface $route_match) {
     $links[] = Link::createFromRoute(t('Home'), '<front>');
     $links[] = Link::fromTextandUrl(t('Health and wellbeing'), Url::fromUri('entity:taxonomy_term/22'));
     $links[] = Link::fromTextandUrl(t('Illnesses and conditions'), Url::fromUri('entity:node/7387'));
-    $links[] = Link::fromTextandUrl(t('A to Z'), Url::fromUserInput('/services/health-conditions-a-z'));
+
+    if ($this->node) {
+      $links[] = Link::fromTextandUrl(t('A to Z'), Url::fromRoute('view.health_conditions.search_page'));
+    }
 
     $breadcrumb->setLinks($links);
     $breadcrumb->addCacheContexts(['url.path']);
 
     // Add cache tags so that if any entities above change, we can regenerate the breadcrumb too.
     $breadcrumb->addCacheTags([
       'taxonomy_term:22',
-      'node:7387',
+      'node:7387'
```

Comment:
Does phpcs not complain about the missing comma?

Reply:
Apparently not! https://www.drupal.org/docs/develop/standards/coding-standards#array

That's crept in from me fiddling with cache tags, so I'll reinstate the trailing comma for consistency.